hera_pspec/pspecdata.py

```diff
@@ -55,16 +57,17 @@ def __init__(self, dsets=[], wgts=None, dsets_std=None, labels=None,
         self.spw_range = None
         self.spw_Nfreqs = None
         self.spw_Ndlys = None
-
+        self.r_params = {}
```

Comment:
r_params is an optional dictionary that stores parameters for any parameteric R matrices that are used. What these parameters are depends on the specific R matrix that is used. In the case of lazy covariance weights, the R matrix is parameterized by a list of filtering window centers, filtering window widths, and the level of filtering to be applied. I have added more documentation.

Reply:
I've added documentation to the line. It would be nice if there were a more appropriate place -- general documentation describing the PspecData class and its parameters. I've added a separate issue to address this.